modules/unauthenticated-users-system-oauth-token-deleter.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-oauth-access-tokens.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-oauth-token-deleter_{context}"]
+= Adding unauthenticated users to the system:oauth-token-deleter role binding
+
+As a cluster administrator, you can add unauthenticated users to the `system:oauth-token-deleter` role binding in {product-title} for specific namespaces. The `system:oauth-token-deleter` role binding allows users to trigger builds from external systems that do not use an {product-title} authentication mechanism. Unauthenticated users do not have access to non-public role bindings by default. This is a change from {product-title} versions before 4.16.
```

Comment:
We would need to phrase this differently. I am not aware of any reason, why someone who is not authenticated should be able to delete Tokens. We would need to say something like:

---

In 4.16 we removed the access from unauthenticated users to the ClusterRole `system:oauth-token-deleter`. If your system relied on this functionality, you can restor it like so:

! IMPORTANT
...
Prerequisites
...
Procedure
...